### PR TITLE
Better report TaskStatus checking

### PR DESCRIPTION
**Pull request by vlagrassa**
- Defines `FINISHED` and `NOT_ERR` as sets of possible `TaskStatus` values
- Check whether a report is finished, etc with method on report itself

### CODE REVIEW

**Comment by vlagrassa** [comment]
Encountered an error when running locally:
`Pipeline Error: EOF occurred in violation of protocol (_ssl.c:2393)`

Closing the PR until I understand this better.

**Comment by vlagrassa** [comment]
...wrong PR

File: src/pkg/caendr/caendr/models/datastore/job_entity.py [hunk 1]
```diff
@@ -104,12 +104,22 @@ def status(self):
...
     self.__status = val
 
 
+  def is_finished(self) -> bool:
+    from caendr.models.task import TaskStatus # Prevents import error
+    return self['status'] in TaskStatus.FINISHED
+
+
+  def is_not_err(self) -> bool:
+    from caendr.models.task import TaskStatus # Prevents import error
```
Comment (r-vieira) [inline src/pkg/caendr/caendr/models/datastore/job_entity.py:119]: let's review this offline (import)

File: src/pkg/caendr/caendr/models/datastore/job_entity.py [hunk 2]
```diff
@@ -104,12 +104,22 @@ def status(self):
...
+    return self['status'] in TaskStatus.NOT_ERR
```
Comment (r-vieira) [inline src/pkg/caendr/caendr/models/datastore/job_entity.py:120]: prefer complete words when possible (NOT_ERROR).

**Review by r-vieira — APPROVED** [review]

**Pull request merged**